server/src/main/xsd/gateway-config-201616.xsd

```diff
@@ -1490,6 +1550,51 @@
         </complexContent>
     </complexType>
 
+	<complexType name="turn.restServicePropertiesType">
+		<complexContent>
+			<restriction base="gateway:ServicePropertiesType">
+				<all>
+					<element maxOccurs="1" minOccurs="0" name="feature.turn.rest"
```

Comment:
This is incorrect, it should use an early access flag to enable this feature.  See https://github.com/kaazing/gateway/blob/193f39243b04f18e9953e1fd2de8ecaa7a47fb81/util/src/main/java/org/kaazing/gateway/util/feature/EarlyAccessFeatures.java